### ISSUE TITLE
SeleniumBase Latest ChromeDriver Installation Fails

### ISSUE THREAD

**Issue by divyac19** [seleniumbase/SeleniumBase#1686]
The command `sbase install chromedriver` results in a failure for ChromeDriver version `109.0.5414.74`. It appears that this version includes two files, the `chromedriver` executable and a file `LICENSE.chromedriver`. The failure seems to occur with both https://chromedriver.storage.googleapis.com/109.0.5414.74/chromedriver_linux64.zip and https://chromedriver.storage.googleapis.com/109.0.5414.74/chromedriver_mac64.zip. Can you please help with this? Please see stack trace below -

```
$ sbase get chromedriver latest

*** chromedriver version for download = 109.0.5414.74 (Latest)

Downloading chromedriver_mac64.zip from:
https://chromedriver.storage.googleapis.com/109.0.5414.74/chromedriver_mac64.zip ...
Download Complete!


* Unable to download driver! Retrying in 3s...


*** chromedriver version for download = 109.0.5414.74 (Latest)

Downloading chromedriver_mac64.zip from:
https://chromedriver.storage.googleapis.com/109.0.5414.74/chromedriver_mac64.zip ...
Download Complete!

** Unable to download driver! Retrying in 5s...


*** chromedriver version for download = 109.0.5414.74 (Latest)

Downloading chromedriver_mac64.zip from:
https://chromedriver.storage.googleapis.com/109.0.5414.74/chromedriver_mac64.zip ...
Download Complete!

Traceback (most recent call last):
  File "/Users/dc/k/ad/s-test/bin/sbase", line 8, in <module>
    sys.exit(main())
  File "/Users/dc/k/ad/s-test/lib/python3.8/site-packages/seleniumbase/console_scripts/run.py", line 977, in main
    sb_install.main()
  File "/Users/dc/k/ad/s-test/lib/python3.8/site-packages/seleniumbase/console_scripts/sb_install.py", line 832, in main
    raise Exception("Expecting only one file in %s!" % zip_file_path)
Exception: Expecting only one file in /Users/dc/k/ad/s-test/lib/python3.8/site-packages/seleniumbase/drivers/chromedriver_mac64.zip!
```

**Comment by mdmintz** [comment]
@divyac19 This was already fixed 3 days ago in ``4.11.4`` - https://github.com/seleniumbase/SeleniumBase/issues/1670#issuecomment-1382683337
(The latest version right now is ``4.11.5``, so just upgrade to that.)

**Comment by mdmintz** [comment]
@divyac19 This was already fixed 3 days ago in ``4.11.4`` - https://github.com/seleniumbase/SeleniumBase/issues/1663
(The latest version right now is ``4.11.5``, so just upgrade to that.)

**Comment by mdmintz** [comment]
@divyac19 This was already fixed 3 days ago in ``4.11.3`` - https://github.com/seleniumbase/SeleniumBase/issues/1663#issuecomment-1381414096
(The latest version right now is ``4.11.5``, so just upgrade to that.)
